### ISSUE TITLE
Islands missing from Scotland (also Farne Islands missing)

### ISSUE THREAD

**Issue by burkmarr** [BiologicalRecordsCentre/vice-counties#5]
This analysis from Andy Amphlett:

With reference to the current GB vice-county boundaries downloaded from BRC's GitHub repository at https://github.com/BiologicalRecordsCentre/vice-counties the following islands are missing from the vc boundary file.

- vc105     202 islands missing
- vc108     372 islands missing

I can supply digitised boundaries for the above as shape files. I used the 'OS Open Map – Local' layer for MLWM to digitise missing islands.  See https://www.ordnancesurvey.co.uk/business-and-government/products/os-open-map-local.html. The digitising was undertaken in 2016 / 2017, using the version of the OS mapping available at that time.

My criteria for digitising was to include any areas above Mean Low Water that were separated from the mainland coastline by open water, and that were not already included in the vice-county boundaries. I did not re-interpret the vice-county boundaries, just added missing islands.

Vice-counties 105 and 108 are the only vice-counties examined in detail. There may be missing islands elsewhere around the Scottish coast.

**Comment by burkmarr** [comment]
Andy has supplied me with the shapefiles of the missing islands.

**Comment by burkmarr** [comment]
Subsequent amendment from Andy:

On 16th Feb., I sent you shapefiles for islands additional to the currently mapped extents of vice-counties 105 and 108. In a fit of enthusiasm (and the weather was grim), I then decided it would be possible to extend my search for 'missing' islands to the whole of mainland Scotland. I have now completed this. Islands were digitised using 'OS Open Zoomstack' as the base mapping in QGIS - https://beta.ordnancesurvey.co.uk/products/os-open-zoomstack. This provides 1:10 000 scale mapping. The definition of an 'island' was any area mapped as 'land' (i.e. above MHWM) along with any surrounding inter-tidal ground above MLWM, and that was separated from the existing v.c. boundary by sea. This definition differs from that I previously used for v.cc 105 and 108, which also included areas which comprised only inter-tidal ground. For Scotland wide consistency, I therefore removed any such areas from the attached files for v.cc 105 and 108, and I recommend that you use these revised files for v.cc 105 and 108. The screenshot below shows an example of my digitising. The red line is the current vc boundary; newly digitised islands have a blue outline; "islands" which only comprise inter-tidal ground were ignored.

![image](https://user-images.githubusercontent.com/9275178/224059032-5fea43cf-d341-4138-bf8e-f0a59c176fcd.png)

I also noticed that the Farne Islands (v.c.68) were missing from the v.c. boundary, so have also digitised them. I have not looked around the rest of the England and Wales coast, and I have not checked for missing islands around the Scottish island vice-counties: v.cc 100, 102, 103, 104, 110, 111 and 112. But from a quick look, there seems to be very few islands missing, so they are a lower priority.  I will probably deal with those areas at a later date.

In summary, additional islands mapped in the attached zip file are:

![image](https://user-images.githubusercontent.com/9275178/224059225-b4da2f6c-bd08-4dfc-a816-b906a9814e35.png)

I have checked the validity of the polygon geometry in each of the files using the tool in QGIS, and all are clean. Any queries, please contact me.

**Comment by burkmarr** [comment]
I did a check (sample) of Andy's digitising, which looked good, and added the islands to the relevant MLWM layer.
